### ISSUE TITLE
Add the Service EPR Field for SOAP Tasks

### ISSUE THREAD

**Issue by praneesha** [wso2/devstudio-tooling-ei#93]
**Description:**
We need to add the Service EPR field for SOAP Tasks. This should be added on the  Main config  tab of the Properties panel for the SOAP task. This is already done for REST tasks.

This field is used to give a Registry location as an EPR of a SOAP task.

i have attached screenshots
![rest-epr](https://user-images.githubusercontent.com/11707273/31178802-c37ff49e-a937-11e7-815d-18297f58c3ad.png)
![soap-no-epr](https://user-images.githubusercontent.com/11707273/31178801-c37067c2-a937-11e7-87e4-bf2dc324fbcd.png)

Please open a DOC issue to update docs once the implementation is complete

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**
https://github.com/wso2/product-ei/issues/934
